uber/configspec.ini

```diff
@@ -337,6 +337,14 @@ group_update_grace_period = integer(default=24)
 # many other registrations (Staff, Dealers, Guests, etc) are in the system.
 max_badge_sales = integer(default=20000)
 
+# Some events require an explicit legal agreement for volunteers as part of
+# the volunteer checklist. For evernts where this is true, set this to true
```

Comment:
evernts

Reply:
ERMAHGERD, MERGFERST IS MAH FAVRIT EVERNT!